bcmc/src/main/java/com/adyen/checkout/bcmc/BcmcComponent.kt

```diff
@@ -154,4 +141,16 @@ class BcmcComponent(
     private fun validateExpiryDate(expiryDate: ExpiryDate): FieldState<ExpiryDate> {
         return CardValidationUtils.validateExpiryDate(expiryDate, Brand.FieldPolicy.REQUIRED)
     }
+
+    companion object {
+        private val TAG = LogUtil.getTag()
+
+        private val PAYMENT_METHOD_TYPES = arrayOf(PaymentMethodTypes.BCMC)
+
+        @JvmField
+        val PROVIDER: PaymentComponentProvider<BcmcComponent, BcmcConfiguration> = BcmcComponentProvider()
+
+        @JvmField
+        val SUPPORTED_CARD_TYPE = CardType.BCMC
```

Comment:
Does this field need to be public?

Reply:
It's being used in the view, I guess it could be `internal`, but it's been like that for quite a while now.